packages/sol/src/constants.ts

```diff
@@ -1,3 +1,47 @@
 export const MINUTES_IN_HOUR = 60;
+export const MINUTES_IN_DAY = 60 * 24;
+export const MINUTES_IN_WEEK = 60 * 24 * 7;
 export const MIN_AGE = 10;
 export const MAX_AGE = 70;
+export const MAX_RULE_DURATION_MINUTES = 8 * 60;
+export const MIN_RULE_DURATION_MINUTES = 0;
```

Comment:
The minimum duration is 15 minutes which is enough to make on short lesson of 15 minutes.